invenio_communities/templates/semantic-ui/invenio_communities/macros/carousel_item.html

```diff
@@ -0,0 +1,43 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2023 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% macro carousel_item(community=None) %}
```

Comment:
I would keep here just the initial macro and add any new code to any instance that implements this e.g zenodo-rdm